tox.ini

```diff
@@ -59,8 +61,7 @@ max-line-length = 100
 
 [gh-actions]
 python =
-  3.7: py37, py37-coverage
-  3.8: py38, py38-plotly, py38-docs, py38-lint
+  3.8: py38, py38-plotly, docs, lint
```

Comment:
I think that the `docs` step could go in another Python version job to make the CI faster since we don't really care about which Python version is required to build the docs.

Reply:
sure

Reply:
we can do this in another PR